workflow.rst

```diff
@@ -190,12 +190,50 @@ The configured property will be used via its implemented getter/setter methods b
             return $this->currentPlace;
         }
 
-        public function setCurrentPlace($currentPlace, $context = []): void
+        public function setCurrentPlace(string $currentPlace, array $context = []): void
         {
             $this->currentPlace = $currentPlace;
         }
     }
 
+.. tip::
```

Comment:
I think we can use a regular paragraph here, it is too long for a tip IMHO

Reply:
You're right, updated :slightly_smiling_face: